x/genutil/utils.go

```diff
@@ -84,6 +84,7 @@ func InitializeNodeValidatorFilesFromMnemonic(config *cfg.Config, mnemonic strin
 	} else {
 		privKey := tmed25519.GenPrivKeyFromSecret([]byte(mnemonic))
 		filePV = privval.NewFilePV(privKey, pvKeyFile, pvStateFile)
+		filePV.Save()
```

Comment:
Is it easy to add a test? If yes, it would be super useful I believe, to avoid regressions.

Reply:
Damn, was this a regression?

Reply:
Gotcha, writing it @amaurym